gstaichi/python/dlpack_funcs.cpp

```diff
@@ -164,11 +164,19 @@ pybind11::capsule field_to_dlpack(Program *program,
   int tree_id = snode->get_snode_tree_id();
   DevicePtr tree_device_ptr = program->get_snode_tree_device_ptr(tree_id);
 
-  int field_in_tree_offset = program->get_field_in_tree_offset(tree_id, snode);
+  int field_in_tree_offset = program->get_field_in_tree_offset(tree_id, snode, /*is_memory_aligned=*/true);
 
   void *raw_ptr = nullptr;
   DLDeviceType device_type = DLDeviceType::kDLCPU;
   std::tie(raw_ptr, device_type) = get_raw_ptr(arch, program, tree_device_ptr);
+
+  void *raw_ptr_check = nullptr;
+  for (int i = 0; i < field_in_tree_offset * 3; i++) {
+    raw_ptr_check = (void *)((uint64_t)raw_ptr + i);
+    int64_t *as_f64_check = reinterpret_cast<int64_t *>(raw_ptr_check);
+    TI_INFO("  offset {} raw as f64: {} {} {}", i, as_f64_check[0], as_f64_check[1], as_f64_check[2]);
```

Comment:
can we remove the TI_INFOs throughout the PR please?
- I think they make the PR harder to read
- they are only needed whilst working on this feature I think?